source/docs/api-reference/api-usage/faults.rst

```diff
@@ -5,8 +5,40 @@ Device Faults
 
 Faults are stored in two fashions. There are "live" faults, which are reported in real-time, and "sticky" faults, which assert persistently and stay asserted until they are manually cleared (like trouble codes in a vehicle).
 
-Sticky Faults can be cleared in Phoenix Tuner X, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
+Sticky Faults can be cleared in Phoenix Tuner X or via the ``clearStickyFaults()`` function on the offending device, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
 
 .. image:: images/self-test-clear-faults.png
    :width: 70%
    :alt: Blink/Clear faults button in tuner located in the bottom right.
+
+Retrieving Faults via API
+-------------------------
+
+Faults can also be retrieved via the API using the ``getFault_XXX()`` (regular) or ``getStickyFault_XXX()`` (sticky) functions on the offending device. ``XXX`` can be replaced with the name of the fault. This can be useful for diagnostics or error handling.
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: java
+
+      .. code-block:: java
+
+         var faulted = cancoder.getFault_BadMagnet().getValue();
```

Comment:
Use `m_cancoder` here and in the C++ example to stay consistent with the rest of the documentation.